src/test/java/com/rocketden/main/service/ProblemServiceTests.java

```diff
@@ -347,7 +344,7 @@ public void getRandomProblemNegativeNumProblems() {
         problem1.setDifficulty(ProblemDifficulty.MEDIUM);
         List<Problem> problems = Collections.singletonList(problem1);
 
-        Mockito.doReturn(problems).when(repository).findAll();
+        //Mockito.doReturn(problems).when(repository).findAll();
```

Comment:
Same as above (I'm not sure why you removed this, so maybe re-examine that - either way, either delete the line or un-comment this out in the code).

Reply:
Same here - just delete that comment?